tools/poll.sh

```diff
@@ -51,7 +50,7 @@ readonly -f get_operation_all
 # MAIN
 
 # Check usage
-if [[ $# -lt 1 ]]; then
+if [[ $# -ne 1 ]] && [[ $# -ne 2 ]]; then
```

Comment:
Hi Matt,

Though the above is fine, but the negation might be hard to read - at least it took me several times.  Maybe the following might be a little simpler:

```Bash
if [[ $# -lt 1 || $# -gt 2 ]] ; then
```

Hope it helps,
~p